src/components/NavBar/nav.css

```diff
@@ -6,63 +6,50 @@
   height: 5rem;
 }
```

Comment:
Love how you were able to fix this problem by streamlining it. The code is easier to read and you have eliminated a lot of redundancies. I wonder if you were able to reason why the original authors decided to go with all of the negative margins.

Reply:
Negative margins is just a common bad practice. I would imagine that the specific reason someone would use it is so that they can align an element without changing any of the parent's styles.